src/collar/oc_dialog.lsl

```diff
@@ -480,7 +480,7 @@ UserCommand(integer iNum, string sStr, key kID) {
 dequeueSensor() {
     //get sStr of first set of sensor details, unpack it and run the apropriate sensor
     //Debug((string)llGetListLength(g_lSensorDetails));
-    list lParams = llParseStringKeepNulls(llList2String(g_lSensorDetails,2), ["|"], []);
+    list lParams = llParseStringKeepNulls(llList2String(g_lSensorDetails,0), ["|"], []);
     //sensor information is encoded in the first 5 fields of the lButtons list, ready to feed to the sensor command,
     list lSensorInfo = llParseStringKeepNulls(llList2String(lParams, 3), ["`"], []);
```

Comment:
shouldnt this be also llList2String(lParams, _1_) instead of llList2String(lParams, _3_) as lParams is build from g_lSensorDetails which has now 2 instead of previous 4 entires?